PersonCell.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  PersonCell.swift
+//  nameToFaces
+//
+//  Created by Felipe Gil on 2021-06-21.
+//
+
+import UIKit
+
+class PersonCell: UICollectionViewCell {
```

Comment:
When we subclass UICollectionViewCell is a good practice to override the prepareForReuse method, the reason lies on how UICollectionView reuses cells for efficiency purposes. See the notes below:

https://developer.apple.com/documentation/uikit/uicollectionreusableview/1620141-prepareforreuse
![image](https://user-images.githubusercontent.com/3866266/124291404-ed395900-db2a-11eb-8b28-01de2b9b3344.png)